sphinx/source/docs/releases/3.0.0.rst

```diff
@@ -82,3 +82,20 @@ Distance Specs
 
 ``ScreenDistanceSpec`` and ``DataDistanceSpec`` are removed. Use ``SizeSpec``
 instead.
+
+``render_mode``
+...............
+
+``render_mode`` was removed from all annotattions. Annotations were cleanly
+separated between those which render to canvas and those that use HTML and CSS.
+The default is canvas rendering. Use the new ``bokeh.models.annotations.html``
```

Comment:
I don't think we should (ever) expose users to sub-module names under `bokeh.models` The current docs now present the module names as categories instead of "modules". This should link to the reference guide section, or just punt and say "Use the new HTML annotations" and give a code snippet

<img width="302" alt="Screen Shot 2021-09-19 at 11 21 16" src="https://user-images.githubusercontent.com/1078448/133938434-408a992b-70d7-44f8-b0b2-c457f7162113.png">
.